app/models/post.rb

```diff
@@ -0,0 +1,3 @@
+class Post < ApplicationRecord
+  belongs_to :user
+end
```

Comment:
- [ ] The ERD diagram shows `author_id` instead of `user_id`. We can solve this issue at the model level to explicitly change the foreign_key column name. We can do this: `belongs_to :author, class_name: 'User'`